kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/RowTrackingSuite.scala

```diff
@@ -231,30 +231,6 @@ class RowTrackingSuite extends DeltaTableWriteSuiteBase with ParquetSuiteBase {
     }
   }
 
-  test("Fail if row tracking is supported but domain metadata is not supported") {
```

Comment:
cc @qiyuandong-db @johanl-db Removing this based on the offline discussion. This scenario is not possible with the new table feature framework.